citesphere-model/src/main/java/edu/asu/diging/citesphere/data/bib/impl/CitationMongoDao.java

```diff
@@ -37,10 +37,20 @@ public class CitationMongoDao implements ICitationDao {
     private MongoTemplate mongoTemplate;
     
     @Override
-    public List<? extends ICitation> findCitations(String groupId, long start, int pageSize, boolean isDeleted, List<String> conceptIds) {
+    public List<? extends ICitation> findCitations(String groupId, long start, int pageSize, boolean isDeleted, List<String> conceptIds, String itemType) {
+        ItemType type = null;
+        if (itemType != null) {
+            type = ItemType.valueOf(itemType);
+        }
         Query query = new Query();
         query.addCriteria(Criteria.where("group").is(groupId));
-        query.addCriteria(Criteria.where("itemType").ne(ItemType.NOTE.name()).andOperator(Criteria.where("itemType").ne(ItemType.ATTACHMENT.name())));
+        if (type == null) {
+            query.addCriteria(Criteria.where("itemType").ne(ItemType.NOTE.name()).andOperator(Criteria.where("itemType").ne(ItemType.ATTACHMENT.name())));
+        } else {
+            query.addCriteria(Criteria.where("itemType").ne(ItemType.NOTE.name()).andOperator(
+                    Criteria.where("itemType").ne(ItemType.ATTACHMENT.name()),
+                    Criteria.where("itemType").is(type.name())));
```

Comment:
if `Criteria.where("itemType").is(type.name())`, `Criteria.where("itemType").ne(ItemType.NOTE.name())` and `Criteria.where("itemType").ne(ItemType.ATTACHMENT.name())` will be true by default.